clients/src/main/java/org/apache/kafka/clients/producer/internals/TransactionManager.java

```diff
@@ -1091,7 +1152,8 @@ private TxnRequestHandler addPartitionsToTransactionHandler() {
 
     private TxnOffsetCommitHandler txnOffsetCommitHandler(TransactionalRequestResult result,
                                                           Map<TopicPartition, OffsetAndMetadata> offsets,
-                                                          ConsumerGroupMetadata groupMetadata) {
+                                                          ConsumerGroupMetadata groupMetadata,
```

Comment:
It seems like there are two very similar methods here. The only difference is the result passed in. I wonder if there is a way we could combine them. (Ie, pass the default result argument)

Reply:
Done.